sigopt/objects.py

```diff
@@ -501,20 +501,54 @@ class Project(ApiObject):
   metadata = Field(Metadata)
 
 
+class Datasets(_DictWrapper):
+  pass
+
+
+class Logs(_DictWrapper):
+  pass
```

Comment:
`TrainingRun.logs` has more structure than this. It has the following json structure:
```json
{
  "logs": {
    "stream1": {
      "content": "log content..."
    },
    "stream2": {
      "content": "log content..."
    }
  }
}
```

In terms of client implementation, I think `run.logs["stdout"].content` or just `runs.logs["stdout"]` are both valid

Reply:
In the current implementation, a user can access logs with `runs.logs["stdout"]["content"]` which is a bit opaque IMO